airflow/dags/rt_loader/external_validation_service_alerts.sql

```diff
@@ -1,38 +1,11 @@
 ---
 operator: operators.SqlQueryOperator
-description: |
-  GTFS RT validation errors as returned by the validator. Each entry corresponds
-  to a <filename>.results.json file that is spit out by the validator.
-fields:
-  errorMessage: |
-    An object with fields for messageId, gtfsRTFeedIterationModel, validationRule,
-    errorDetails.
-  occurrenceList: |
-    An array of objects with these fields: occurenceId, messageLogModel, prefix
+dependencies:
+    - load_rt_validations
 ---
 
-CREATE OR REPLACE EXTERNAL TABLE gtfs_rt.validation_service_alerts (
-    errorMessage STRUCT<
-      messageId INT64,
-      gtfsRTFeedIterationModel STRING,
-      validationRule STRUCT<
-        errorId STRING,
-        severity STRING,
-        title STRING,
-        errorDescription STRING,
-        occurrenceSuffix STRING
-      >,
-      errorDetails STRING
-    >,
-    occurrenceList ARRAY<
-      STRUCT<
-        occurrenceId INT64,
-        messageLogModel STRING,
-        prefix STRING
-      >
-    >
-)
+CREATE OR REPLACE EXTERNAL TABLE gtfs_rt.validation_service_alerts
 OPTIONS (
-    uris=["gs://calitp-py-ci/gtfs-rt-validator-api/test_output_full/*gtfs_rt_service_alerts_url"],
-    format="JSON"
+    uris=["{{get_bucket()}}/rt-processed/validation/*/service_alerts.parquet"],
```

Comment:
Just confirming -- we to this point have not been loading anything, right? All the validations that are loaded are "test" data for `126` and `106` only? Do we want to run a backfill on this at some point (immediately on merge)?

Reply:
I think so, the existing job writes to a different path entirely.

A backfill will be possible, we just need to decide how far back we want to go.